installer/Migration/V5_2_0/Migration.php

```diff
@@ -172,12 +173,23 @@ public function up(): void
         $this->getConfigHelper()->deleteConfigValue('hsp_used_last_updated');
     }
 
-    /**
-     * @inheritDoc
-     */
-    public function getVersion(): string
+    private function cleanLDAPAddonData(): void
```

Comment:
seems function order also changed. hard to understand, better if you can revert those changes